bin/mail/2016-09-15-latam-hoc-starwars/generate-mailing-lists.rb

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# Query all HOC organizers and Code Studio teachers from any of the Latam countries.
+
+# Latin America (latam)
+LATAM_ES_COUNTRIES = [
+  'Mexico',
+  'Guatemala',
+  'El Salvador',
+  'Costa Rica',
+  'Nicaragua',
+  'Belize',
+  'Panama',
+  'Cuba',
+  'Dominican Republic',
+  'Venezuela',
+  'Colombia',
+  'Ecuador',
+  'Peru',
+  'Bolivia',
+  'Paraguay',
+  'Uruguay',
+  'Chile',
+  'Argentina',
+  'Puerto Rico'
```

Comment:
This list was derived by Kate and our LATAM partners last year. They were probably targeting where we had partners and highest population. Our campaign defines LATAM as all Spanish speaking countries in North/Central/South America + Caribbean so let's just use all the countries.

[Here](http://lanic.utexas.edu/subject/countries/) might be an easier to parse list.